mev_inspect/inspect_block.py

```diff
@@ -104,7 +106,8 @@ def get_checksum_address(token):
     """
     tries to return the checksum of address, if it fails just return the address
     """
-    token = str(token).replace(TRAILING_ZEROS, "")
+    if token not in [WETH_OP, OP]:
```

Comment:
Let's take the time to fix this issue in a more general way.

Reply:
Done